src/yang/ietf-qos-oper.yang

```diff
@@ -141,10 +141,10 @@ module ietf-qos-oper {
         "Bytes of packets dropped by meter";
     }
   }
-  grouping classifier-entry-statistics {
+  grouping classifier-statistics {
```

Comment:
Since this is the oper model, and almost everything is about statistics, why even have the word statistics in the name?

Reply:
This is just grouping.. it will not be part of the tree.. also, it is used under "list classifier", so would look strange to use "classifier". I can change to "classifier-counters" if that would look better..

Reply:
In that case, ignore my comment.